### PR TITLE
Feat: added trimStart under the string manipulation

### PR DESCRIPTION
**Pull request by GIHAA**
#46

### CODE REVIEW

File: packages/flodash/lib/modules/string/main.dart [hunk 1]
```diff
@@ -9,3 +9,4 @@ export 'replace.dart';
 export 'upper_first.dart';
 export 'unescape.dart';
 export 'uppercase.dart';
+export 'trimStart.dart';
```
Comment (Akalanka47000) [inline packages/flodash/lib/modules/string/main.dart:12]: Shall we switch this to snakecase ?

File: packages/flodash/test/modules/string/trimStart_test.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import 'package:flodash/flodash.dart' as flodash;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('capitalize letters with spaces', () {
+    String str = flodash.camelCase('Foo Bar');
```
Comment (Akalanka47000) [inline packages/flodash/test/modules/string/trimStart_test.dart:6]: These tests are calling the camelCase function ne, something looks wrong

**Review by Akalanka47000 — CHANGES_REQUESTED** [review]

File: packages/flodash/lib/modules/string/main.dart (hunk 1 above)
Comment (GIHAA) [reply packages/flodash/lib/modules/string/main.dart]: okay

**Comment by Akalanka47000** [comment]
Cherry picked the relevant commits

**Pull request closed without merging**